api/src/main/java/cz/muni/fi/facade/DriverFacade.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.muni.fi.facade;
+
+import cz.muni.fi.dto.DriverCreateDTO;
+import cz.muni.fi.dto.DriverDTO;
+import java.util.List;
+
+/**
+ * @author Lucie Kurečková, 445264
+ */
+public interface DriverFacade {
+    /**
+     * Find all drivers
+     * 
+     * @return list of DriverDTO
+     */
+    public List<DriverDTO> getAllDrivers();
```

Comment:
pls remove `public` from interface methods. reasons here - https://stackoverflow.com/questions/161633/should-methods-in-a-java-interface-be-declared-with-or-without-a-public-access-m